eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala

```diff
@@ -80,7 +80,7 @@ object ChannelRelayer {
               case Some(nextNodeId) => node2channels.get(nextNodeId).map(channelUpdates).map(c => c.channelUpdate.shortChannelId -> c).toMap
               case None => Map.empty
             }
-            context.log.debug(s"spawning a new handler with relayId=$relayId to nextNodeId={} with channels={}", nextNodeId_opt.getOrElse(""), channels.keys.mkString(","))
+            context.log.info(s"spawning a new handler with relayId=$relayId to nextNodeId={} with channels={}", nextNodeId_opt.getOrElse(""), channels.keys.mkString(","))
```

Comment:
This one feels to me like a `debug` log, why not activate `debug` logs through logback config only for the `ChannelRelayer` if you're investigating an issue?

Reply:
Setting the log level to `DEBUG` gets pretty spammy because of the `channel_update`s events.

Anyway those logs are less useful after f264235637feb9dd8817daf0c5b706cf887747f3, I'll simply revert.

Reply:
Done with e4270fc29.